cmd/soroban-cli/src/commands/contract/mod.rs

```diff
@@ -165,3 +174,16 @@ pub enum SpecOutput {
     /// Pretty print of contract spec entries
     Docs,
 }
+
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, clap::ValueEnum, Default)]
+pub enum InfoOutput {
+    /// Pretty-printed output of the info entry
+    #[default]
+    Pretty,
+    /// XDR output of the info entry
+    XdrBase64,
+    /// JSON output of the info entry (one line, not formatted)
+    Json,
+    /// Formatted (multiline) JSON output of the info entry
+    JsonFormatted,
+}
```

Comment:
Can we keep this type in the info shared module? It seems used primarily there. If it ever gets used elsewhere we can move it to a suitable place to be shared with more things then.

Reply:
I was not sure if it needs to be in this particular place or not and followed how another selection enums are structured. We should probably move them out as well in a separate PR